### ISSUE TITLE
Possible to use comments with hyphens or at least have this extension recognize {%- comment -%} blocks as comments

### ISSUE THREAD

**Issue by t3pfaffe** [Christopher-Hayes/liquid-comments#2]
Currently the extension only prints the comment block without hyphens `{% comment %}`, can an option be added to use `{%- comment -%}` blocks as well?

Additionally with this extension, VSCode seems to not realize lines of code that are already wrapped in `{%- comment -%} ... {%- endcomment -%}` are already commented resulting in VSCode appending the comment blocks on it again (ex. `{% comment %} {%- comment -%} ... {%- endcomment -%} {% endcomment %}`).

**Comment by Christopher-Hayes** [comment]
Good point, I'm onboard with switching to hyphens. For the second issue, might be able to get it to recognize both, not sure. I'll try to get that added and pushed to VSCode ext store. Or if you'd like to try making that code change and starting a pull request, the repo isn't too complicated.
